target/linux/generic/files/drivers/net/phy/rtk/Makefile

```diff
@@ -0,0 +1,66 @@
+obj-$(CONFIG_RTL8367S_PHY) += rtl8367s_gsw.o
```

Comment:
New files should contains an SPDX license identifier.

Reply:
These files are actually from `target/linux/mediatek/files-5.4/drivers/net/phy/rtk/` if that makes any difference.
Should the license IDs be on all of the new files on the PR (~125, excluding patches)?
Or an ID for this Makefile would be enough, since there's a slight chance of code?